File: __tests__/__fixtures__/output/ruby/native/query-encoded.rb
require 'uri'
require 'net/http'
require 'openssl'

url = URI("https://httpbin.org/anything?startTime=2019-06-13T19%3A08%3A25.455Z&endTime=2015-09-15T14%3A00%3A12-04%3A00")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body

File: __tests__/__fixtures__/output/node/request/query-encoded.js
const request = require('request');

const options = {
  method: 'GET',
  url: 'https://httpbin.org/anything?startTime=2019-06-13T19%3A08%3A25.455Z&endTime=2015-09-15T14%3A00%3A12-04%3A00'
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

